setup.py

```diff
@@ -1,9 +1,9 @@
 from distutils.core import setup, Extension
 from pip.req import parse_requirements
-install_reqs = parse_requirements('requirements.txt', session=False)
+# install_reqs = parse_requirements('requirements.txt', session=False)
```

Comment:
It's been awhile since I looked at setuptools stuff. I think @tavianator was looking into pip installing some complicated stuff for another project and he also likes Python 3. @tavianator do you know what to do for `install_reqs` ?